### DISCUSSION TITLE
QL crashes on some home-split FLAC files

### DISCUSSION THREAD

**Discussion by EricViala** [quodlibet/quodlibet#4196] [Q&A] [answered]
I got [this wonderfull][1], semi-legal recording of some Mozart sonatas, all ripped into one big fat FLAC file without any playlist or metadata. As I like to know what I'm listening, I painstakingly split the file on the command line, e.g.

`flac --skip 48:22.00 --until 57:48.00 -o K511.flac -f pianosonata11inm00moza.flac`

I then tagged the ten resulting files with [Ex Falso][2].

All that plays nice on Rythmbox, VLC and Audacious. On Quod Libet, which I prefer, the tracks are correctly displayed, any attempt to play them however results in immediate crash.

The original [`pianosonata11inm00moza.flac`][3] plays nice on Quod Libet.

Below is the error that is reported, not very helpfull I'd suppose.

    FaultHandlerCrash: Fatal Python error: Segmentation fault
    ------
    Traceback (most recent call last):

      File "/usr/lib/python3/dist-packages/quodlibet/errorreport/faulthandling.py", line 138, in raise_and_clear_error
        raise FaultHandlerCrash(text)

    quodlibet.errorreport.faulthandling.FaultHandlerCrash: Fatal Python error: Segmentation fault

If this of any relevance, I'm on Debian 11 with XFCE desktop.

  [1]: https://archive.org/details/pianosonata11inm00moza
  [2]: https://quodlibet.readthedocs.io/en/latest/
  [3]: https://archive.org/download/pianosonata11inm00moza/pianosonata11inm00moza.flac

**Comment by declension** [comment]
Thanks. What audio system are you using? i.e. is it pulseaudio or are you now on Pipewire? 

FWIW I've noticed problems with seeking in flac files recently (TODO: create issue), maybe similar timings to switching to Pipewire (and Manjaro), but maybe not (TODO: replicate on Ubuntu)

**Reply by declension** [reply, marked as answer]
@EricViala FWIW that file plays OK for me (other than the seeking issue mentioned, as now added in #4197)
